src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -867,39 +867,17 @@ length(title) < 5
 
 == Page Block Magnification
 
-Page blocks have a magnification option that allows the user to zoom in order to inspect the details of the block. 
-This is useful because it allows the user to examine the page block in distraction-free mode and focus on that specific block in particular.
-
-=== Magnification Options
-
-.Page Block Magnification Options:
-[cols="1s,5a"]
-|===
-
-| [#tpl-syntax-fref-disabled]#<<tpl-syntax-fref-disabled,Disabled>>#
-|
-Block magnification is disabled.
-
-| [#tpl-syntax-fref-modal]#<<tpl-syntax-fref-modal,Modal>>#
-|
-The block magnification is in the form of a modal.
-
-| [#tpl-syntax-fref-fullscreen]#<<tpl-syntax-fref-fullscreen,Fullscreen>>#
-|
-The block magnification is in the form of a full-width and full-height modal.
-
-|===
-
-image:compose-configuration/enable-magnification.png[Page Block Magnification Options, role="data-zoomable"]
-
-=== Enabling Page Block Magnification
-
+Page blocks have a magnification option that allows the user to zoom in order to inspect the details of the block in distraction-free mode. 
 By default, block magnification is disabled. 
-When set, blocks that have this configuration will have a zoom-in icon in the top right corner of the block, allowing the user to magnify the block upon clicking on the icon. 
+When set, blocks that have this configuration will have a zoom-in icon in the top right corner of the block, allowing the user to magnify the block upon clicking on the icon.
+
```

Comment:
I'd consider moving the image where you outline the zoom button here (instead of under the config section).
To me, it fits better here.